src/web/App.scss

```diff
@@ -36,7 +36,7 @@
   }
 
   button {
-    border: 1px solid var(--theme-button-border);
```

Comment:
Setting the border to the main button color variable doesn't really make sense. Just change the --theme-button-border color to be the same.